ckanext/datavic_iar_theme/webassets/_datavic_autocomplete.js

```diff
@@ -23,6 +23,22 @@ ckan.module("-datavic-autocomplete", function ($) {
       newRole
     );
       target.attr("role", newRole);
-      target.attr("aria-expanded", "false")
+      target.attr("aria-expanded", "false");
+      target.prop("required", true);
```

Comment:
Added necessary attributes addition functionality to dynamically changed select2 fields